client/src/components/geneExpression/index.tsx

```diff
@@ -180,8 +180,10 @@ class GeneExpression extends React.Component<{}, State> {
         style={{
           display: "flex",
           flexDirection: "column",
-          justifyContent: "space-between",
+          justifyContent: "start",
           padding: globals.rightSidebarSectionPadding,
+          height: "50%",
```

Comment:
```suggestion
          min-height: "50%",
```

Otherwise, if the drawer isn't open, the continuous part is still only able to grow to 50% of the height